packages/deployment/app/config.yml

```diff
@@ -120,6 +120,17 @@ default:
               - Arn
           protocol: lambda
 
+    # This topic receives errored kinesis messages and tries them again.
+    kinesisFallback:
```

Comment:
Should this be in the core config or in the example/app/config?

Reply:
I think it needs to be in core config since all cumulus deployments would now come with fallbackConsumer